frontend/src/app/main/ui/dashboard/projects.cljs

```diff
@@ -51,10 +52,88 @@
       [:div.info
        [:span (tr "dasboard.team-hero.text")]
        [:a {:on-click  go-members} (tr "dasboard.team-hero.management")]]]
-     [:button.invite {:on-click invite-member} (tr "onboarding.choice.team-up.invite-members")]
+     [:button.btn-primary.invite {:on-click invite-member} (tr "onboarding.choice.team-up.invite-members")]
      [:button.close {:on-click close-banner}
       [:span i/close]]]))
 
+(def builtin-templates
+  (l/derived :builtin-templates st/state))
+
+(mf/defc tutorial-project
+  [{:keys [close-tutorial default-project-id] :as props}]
+  (let [state (mf/use-state
+               {:status :waiting
+                :file nil})
+
+        template (->>  (mf/deref builtin-templates)
+                       (filter #(= (:id %) "tutorial-for-beginners"))
+                       first)
+
+        on-template-cloned-success
+        (mf/use-callback
+         (fn [response]
+
+           (swap! state
+                  (fn [state]
+                    (-> state
+                        (assoc :status :success :file (:first response)))))
```

Comment:
perhaps this is better as an one liner
```clojure
(swap! state #(assoc % :status :success :file (:first response)))
```

(or in two lines, one with :status and other with :file)